Makefile

```diff
@@ -0,0 +1,9 @@
+help: ## Print documentation
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+ghcid-devel: ## Run the server in fast development mode. See DevelMain for details.
+	ghcid \
+	    --command "stack ghci servant-persistent" \
+	    --test "DevelMain.update"
+
+.PHONY: ghcid-devel help
```

Comment:
Every project should have a `Makefile`! They're so useful for providing short little commands that you'll want to use frequently.